Backpack-SwiftUI/CardList/Classes/BPKCardListSectionHeaderAction.swift

```diff
@@ -0,0 +1,37 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import SwiftUI
+
+public extension BPKCardListLayout {
+    struct SectionHeaderAction {
+        let icon: BPKIcon
```

Comment:
Should this use the BPKButton.Icon like BPKCardListAccessoryAction

Reply:
Personally, even though it looks weird that they're different, I think it makes more sense like this.
SectionHeader button by design shouldn't allow text, only icons (that came up as a requirement after it was contributed so not implemented yet), and therefore using `BPKButton.Icon` is rather pointless since both leading and trailing positions would be the same. We would be ignoring it since we should only use the `BPKButton` init without title

The accessory button is more flexible though, and could have both title, and icon in any position